cmd/cadvisor.go

```diff
@@ -77,6 +78,8 @@ var rawCgroupPrefixWhiteList = flag.String("raw_cgroup_prefix_whitelist", "", "A
 
 var perfEvents = flag.String("perf_events_config", "", "Path to a JSON file containing configuration of perf events to measure. Empty value disabled perf events measuring.")
 
+var traceEndpoint = flag.String("trace_endpoint", "", "Url host:port to the OTLP collector")
```

Comment:
why are you reading this variable here? Would it be better to hide this logic inside the `InitTrace`?